packages/appstore/src/components/product-card/product-card.scss

```diff
@@ -0,0 +1,135 @@
+.product-card {
+    width: 200px;
+    height: 120px;
+    border-radius: 8px;
+    overflow: hidden;
+
+    @include mobile {
+        width: 144px;
+        height: 96px;
+    }
+
+    &__trade-type {
+        position: relative;
+
+        &--body {
```

Comment:
@njazuli-deriv Modifier pattern is used for the `body` but it's an element. Modifier patterns should be used only for flags in an element. Please check http://getbem.com/naming/
For this, you can use single dash `-body`.